tests/psoc6/i2c_soft.py

```diff
@@ -0,0 +1,9 @@
+#### SoftI2C
+import machine
```

Comment:
Not required to import the whole of `machine` module. Handled by the next import statement

Reply:
right